salt/modules/dockerng.py

```diff
@@ -1555,11 +1591,16 @@ def _valid_network_mode():  # pylint: disable=unused-variable
                 # Ensure that the user didn't just pass 'container:', because
                 # that would be invalid.
                 return
-            raise SaltInvocationError()
+            else:
+                # just a name assume it is a network
+                log.info(
+                    'Assuming network_mode {0!r} is a network.'.format(
```

Comment:
No problem, it's quite prevalent in the codebase, so it's an easy thing to fall into doing.